browser/src/map/handler/Map.Keyboard.js

```diff
@@ -436,6 +436,37 @@ L.Map.Keyboard = L.Handler.extend({
 				this._map._docLayer._preview.partsFocused = false;
 			}
 		}
+		else if (this._isCtrlKey(ev) && !ev.shiftKey && !ev.altKey && ev.keyCode === this.keyCodes.F) {
+			if (app.UI.language.fromURL === 'de' && this._map.getDocType() === 'text') {
+				this._map.sendUnoCommand('.uno:Navigator');
+			}
+			else {
+				if (!this._map.uiManager.isStatusBarVisible()) {
+					this._map.uiManager.showStatusBar();
+				}
+				this._map.fire('focussearch');
+			}
+
+			ev.preventDefault();
+		}
+		else if (this._isCtrlKey(ev) && !ev.shiftKey && ev.altKey && ev.keyCode === this.keyCodes.S && app.UI.language.fromURL === 'de' && this._map.getDocType() === 'text') {
+			this._map.fire('focussearch');
+			ev.preventDefault();
+		}
+		else if (this._isCtrlKey(ev) && ev.keyCode === this.keyCodes.S) {
```

Comment:
this one can be leaved as it is as it is very important one and language-neutral